web/src/message_list.ts

```diff
@@ -540,7 +540,6 @@ export class MessageList {
     show_edit_topic_on_recipient_row($recipient_row: JQuery, $form: JQuery): void {
         $recipient_row.find(".topic_edit_form").append($form);
         $recipient_row.find(".on_hover_topic_edit").hide();
-        $recipient_row.find(".edit_message_button").hide();
```

Comment:
I think action buttons are not hidden on the floating/top recipient bar in a conversation view; does this handle those properly?

Reply:
If I'm understanding you correctly, I believe so. The only place `.edit_message_button` appears is in the action buttons. There's no such class in the recipient bar where the edit-topic functionality is.

Reply:
Ahh, good point.